website/docs/docs/mesh/iceberg/bigquery-iceberg-support.md

```diff
@@ -73,25 +101,134 @@ select * from {{ ref('jaffle_shop_customers') }}
+
+The available adapter properties for configuration are `base_location_root`, `base_location_subpath`, and `storage_uri`. `base_location_subpath` and `storage_uri` are only accepted in model configurations (see [Adapter Properties](#adapter-properties)).
+
+- If no inputs are provided, dbt will output for base_location `{{ external_volume }}/_dbt/{{ schema }}/{{ model_name }}`
+- If base_location_root = `foo`, dbt will output `{{ external_volume }}/foo/{{ schema }}/{{ model_name }}`
+- If base_location_subpath = `bar`, dbt will output `{{ external_volume }}/_dbt/{{ schema }}/{{ model_name }}/bar`
+- If base_location_root = `foo` and base_location_subpath = `bar`, dbt will output `{{ external_volume }}/foo/{{ schema }}/{{ model_name }}/bar`
+
+A theoretical (but not recommended) use case is re-using the `storage_uri` while maintaining isolation across development and production environments. We recommend against this as storage permissions should be configured on the external volume and underlying storage, not paths that any analytics engineer can modify.
+
+dbt also allows users to completely override the storage_uri with the adapter property `storage_uri`. This overrides both the catalog integration path and any `base_location` overrides to supply the entire `storage_uri` path directly.
+
+#### Example configurations
+
+An example model with a customized `base_location`:
+
+<File name='iceberg_model.sql'>
+
+```sql
+
+{{
+    config(
+        materialized='table',
+        catalog_name='my_bigquery_iceberg_catalog',
+        adapter_properties={
+          'base_location_root': 'foo',
+          'base_location_subpath': 'bar',
+        },
+
+    )
+}}
+
+select * from {{ ref('jaffle_shop_customers') }}
+```
+
+</File>
+
+An example `catalogs.yml` with a customized `base_location_root`:
+
+<File name='catalogs.yml'>
+
+```yaml
+catalogs:
+  - name: my_bigquery_iceberg_catalog
+    active_write_integration: biglake_metastore
+    write_integrations:
+      - name: biglake_metastore
+        external_volume: 'gs://mydbtbucket'
+        table_format: iceberg
+        file_format: parquet
+        catalog_type: biglake_metastore
+        adapter_properties:
+          base_location_root: foo
+```
+
+</File>
+
+:::info Legacy model configuration for base_location
+
+For backwards compatibility, dbt <Constant name="fusion"/> also supports setting `base_location` properties and `storage_uri` as top-level model configuration fields. `adapter_properties` configs take precedence over legacy configs.
+
+For example, in the following model config, `base_location_root`=`bar` overrides `base_location_root`=`foo`.
```

Comment:
this is a great example, t hank you!

Reply:
@ajhlee-dbt do you think this would be a good blurb to add in the [adapter properties](https://docs-getdbt-com-git-ajhlee-bigquery-iceberg-revision-dbt-labs.vercel.app/docs/mesh/iceberg/bigquery-iceberg-support#adapter-properties) section too? i know we say 'go to base location for more info' but might be also good to add?

> base_location properties and storage_uri as top-level model configuration fields. adapter_properties configs take precedence over legacy configs.